pyproject.toml

```diff
@@ -34,7 +34,11 @@ branch = true
 # Required to get full coverage when using Playwright
 concurrency = ["greenlet", "thread"]
 plugins = ["django_coverage_plugin"]
-omit = ["*/assets/*"]
+omit = [
+  "*/assets/*",
+  # remove this when we have a UI to view audit logs
+  "airlock/management/commands/audit.py",
```

Comment:
I don't think testing this would be hard!

Reply:
I was concerned that diff had grown too large already, and I only really added to this to for a post-deploy check in prod.

Reply:
tests added

Reply:
Fine.